examples/sync/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "commonware-sync"
+edition = "2021"
+publish = true
+version = "0.0.54"
+license = "MIT OR Apache-2.0"
+description = "Sync a new [adb::any::Any] database to an existing instance's state."
```

Comment:
In this description, we should leave links out IMO.

Let's do: `Synchronize state between a server and client.`